sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_etl_sql_run_check_v1/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Bigquery ETL SQL Run Check
+description: |-
+  This query is scheduled frequently and used to detect any breakages in the
+  bigquery-etl tooling as soon as possible to ensure nightly ETL tasks can run.
+owners:
+- ascholtz@mozilla.com
+labels:
+  incremental: false
+scheduling:
+  dag_name: bqetl_status_check
+  date_partition_parameter: null
```

Comment:
Might be worth a comment here explaining we're intentionally overwriting the entire table each time and keeping just the latest record.